feature/system/management/README.md

```diff
@@ -0,0 +1,146 @@
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/addresses/address/config/ip
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/addresses/address/config/prefix-length
+*   Configure the loopback interface to participate in the VRF "mgmt"
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/vrf/name = mgmt
+
+##### Configure linecard ports to ATE using BGP
+
+*   Configure IPv6 addresses on DUT and ATE ports 1 and 2. Configure them to participate in the VRF "mgmt"
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/addresses/address/config/ip
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/addresses/address/config/prefix-length
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/vrf/name = mgmt
+*   Configure IPv6 eBGP between DUT Port-1 <--> ATE Port-1 and DUT Port-2 <--> ATE Port-2 in VRF "mgmt"
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/global/config/as = 64512
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/global/config/router-id = <router_id>
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=1<neighbor_ip>]/config/peer-as = 64511
+    * /network-instances/network-instance[name=mgmt]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=<neigh_ip>]/afi-safis/afi-safi[afi-safi-name=ipv6-unicast]/config/route-distinguisher = 64512:100 
+*   Set default import and export policy to ```ACCEPT_ROUTE``` for the eBGP sessions
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/default-import-policy
+    *   /network-instances/network-instance[name=mgmt]/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/apply-policy/config/default-export-policy
+*   Advertise a default route from ATE to DUT throught both the BGP sessions
+*   Advertise the loopback interface from DUT to ATE through both the BGP sessions
+    *   /network-instances/network-instance[name=default]/protocols/protocol[identifier=BGP, name=BGP]/neighbor[ip=<neigh_ip>]/afi-safis/afi-safi[afi-safi-name=ipv6-unicast]/config/prefix = <lo0 IPv6 Address>
+
+##### Configure linecard ports to ATE using VRRP
+
+*   Configure IPv6 addresses on DUT and ATE ports 3 and 4 from one IPv6 subnet ```vrrp_subnet```
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/addresses/address/config/ip
+    *   /interfaces/interface/subinterfaces/subinterface/ipv6/addresses/address/config/prefix-length
+*   Configure ATE Port-3 and Port-4 to run VRRP
+    *   Configure a VRRP group-1 with a virtual IP address from the ```vrrp_subnet```, priority 100 and preempt enabled on ATE Port-3
+    *   Configure a VRRP group-1 with the same virtual IP address, priority 120 and preempt enabled on ATE Port-4
```

Comment:
The usage of VRRP assumes that there's a shared layer-2 segment between the ports. How is it going to be achieved in the context of the test?

Is the ATE device supposed to emulate the shared segment?